services/bigpanda/service.go

```diff
@@ -83,20 +83,24 @@ func (s *Service) StateChangesOnly() bool {
 }
 
 type testOptions struct {
-	AppKey    string          `json:"app_key"`
-	Message   string          `json:"message"`
-	Level     alert.Level     `json:"level"`
-	Data      alert.EventData `json:"event_data"`
-	Timestamp time.Time       `json:"timestamp"`
+	AppKey            string          `json:"app_key"`
+	Message           string          `json:"message"`
+	Level             alert.Level     `json:"level"`
+	Data              alert.EventData `json:"event_data"`
+	Timestamp         time.Time       `json:"timestamp"`
+	PrimaryProperty   string          `json:"primary_property"`
+	SecondaryProperty string          `json:"secondary_property"`
 }
 
 func (s *Service) TestOptions() interface{} {
 	t, _ := time.Parse(time.RFC3339, "1970-01-01T00:00:01Z")
 
 	return &testOptions{
-		AppKey:  "my-app-key-123456",
-		Message: "test bigpanda message",
-		Level:   alert.Critical,
+		AppKey:            "my-app-key-123456",
+		Message:           "test bigpanda message",
+		Level:             alert.Critical,
+		PrimaryProperty:   "test_property1",
+		SecondaryProperty: "test_property2",
```

Comment:
```suggestion
```

The workflow at https://docs.influxdata.com/kapacitor/v1.5/working/api/#testing-a-service uses this method to return defaults for testing. Both primary and secondary properties are optional, so they can be empty (omitted) as they are by default in TICKScript, unless these values help with testing. The `testOptions` struct in this PR , when serialized to JSON, already indicates (in this PR) that these properties can be set.